docs/src/pages/components/data-grid/group-pivot/group-pivot.md

```diff
@@ -7,39 +7,39 @@ components: DataGrid, XGrid
 
 <p class="description">Use grouping, pivoting and more to analyse the data in depth.</p>
 
-## 🚧 Tree data [<span role="img" title="Enterprise">⚡️</span>](https://material-ui.com/store/items/material-ui-x/)
+## 🚧 Tree data [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-x/)
```

Comment:
Just a note, maybe it is just me but I found it a bit difficult to differentiate the Pro from the Premium icon as the only difference is the color (and they are both blue, just different shades of it).

Reply:
The shares are also different, premium is longer.